desktop_preact/Banner.jsx

```diff
@@ -63,6 +63,10 @@ export class Banner extends Component {
 		}
 	}
 
+	onFinishedTransitioning = () => {
+		this.props.onFinishedTransitioning();
```

Comment:
This is just a suggestion, that goes against the "You should not use comments as version control" dogma, but might help our future selves to determine where to put the progress bar start. But maybe we're clever enough to figure that out ...

```suggestion
		this.props.onFinishedTransitioning();
		// Uncomment when we have a progress bar during the campaign again
		// this.startProgressbar()
```

Reply:
I also added the register progress bar callback method back in